dojo/settings/settings.dist.py

```diff
@@ -1744,6 +1748,15 @@ def saml2_attrib_map_format(dict):
 # see https://github.com/laymonage/django-jsonfield-backport
 SILENCED_SYSTEM_CHECKS = ["django_jsonfield_backport.W001"]
 
+# Problem utilizes mappings from a JSON file to disambiguate Findings.
+# The JSON file should have the following structure:
+# {
+#     "problem_id_1": ["script_id_1", "script_id_2", "script_id_3"],
+#     "problem_id_2": ["script_id_4", "script_id_5"],
+#     "problem_id_3": ["script_id_6"]
+# }
+PROBLEM_MAPPINGS_JSON_URL = "https://homepages.dcc.ufmg.br/~leonardooliveira/disambiguator.json"
```

Comment:
Maybe move this to the comment, explain it is a mapping that aggregates equivalent/similar `Findings` from Nmap, OpenVAS, Metasploit and Nuclei. Have an accompanying `README.md` file explaining how the mapping is built. Set the default configuration to `None` to make it opt-in.

Also, if this variable is set to `None`, it would be best to simply hide the "Problems" button in Dojo's left toolbar. (So it doesn't cause any changes to deployments unless the mapping is configured.)